tests/unit/specs/components/network/NetworkList.spec.js

```diff
@@ -68,4 +71,14 @@ describe(`NetworkList`, () => {
     wrapper.find(".select-network-item.selected").trigger("click")
     expect(wrapper.vm.$store.dispatch).not.toHaveBeenCalledWith()
   })
+
+  it(`shows windows about switching networks`, async () => {
+    wrapper.setData({
```

Comment:
This test is not doing what it should. I guess you want to mock the window object and check if the confirmation got triggered.